Makefile

```diff
@@ -74,6 +74,7 @@ nodejs_sdk::
 	$(WORKING_DIR)/bin/$(CODEGEN) -version=${VERSION} nodejs $(SCHEMA_FILE) $(CURDIR)
 	cd ${PACKDIR}/nodejs/ && \
 		echo "module fake_nodejs_module // Exclude this directory from Go tools\n\ngo 1.17" > go.mod && \
+		rm -rf node_modules yarn.lock && \
```

Comment:
Perhaps we should only just remove the lockfile? This would reduce the number of times we fetch node packages.